src/preamble.js

```diff
@@ -1822,6 +1822,7 @@ function removeRunDependency(id) {
 
 Module["preloadedImages"] = {}; // maps url to image data
 Module["preloadedAudios"] = {}; // maps url to audio data
+Module["preloadedWasm"] = {}; // maps url to wasm instance exports
```

Comment:
please put this behind the WASM and MAIN_MODULE flags

(also, we should probably put the others behind a flag for preload-plugins, not sure if we have one yet)